build.rs

```diff
@@ -1,5 +1,19 @@
 // generated by `sqlx migrate build-script`
-fn main() {
+fn main() -> std::io::Result<()> {
     // trigger recompilation when a new migration is added
-    println!("cargo:rerun-if-changed=migrations");
+    // println!("cargo:rerun-if-changed=migrations");
```

Comment:
please revert => don't remove the recompilation trigger when a new migration script is added